src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -123,11 +123,19 @@ public static void propagateIgnoreGlobList(Author author, List<String> ignoreGlo
         author.appendIgnoreGlobList(ignoreGlobList);
     }
 
+    /**
+     * Adds author to the {@code AuthorList}.
+     */
     public void addAuthor(Author author) {
         authorList.add(author);
         setAuthorDetails(author);
     }
 
+    /**
+     * Overloads {@code addAuthor} and {@code propagateIgnoreGlobList} methods.
```

Comment:
Actually what I meant is like this, which is the common javadocs format for overload methods.
```
Adds {@code author} to the {@code AuthorList}, and propagates {@code ignoreGlobList} to the {@code author}.